shoplist/src/main/java/com/netguru/codereview/ui/model/ShopList.kt

```diff
@@ -0,0 +1,11 @@
+package com.netguru.codereview.ui.model
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+
+class ShopList(
```

Comment:
- This class only holds data so it should be data class
- Currently only iconUrl variable is used. Will other variables be needed later? If not, remove them